Documentation/specs/runtime-configuration-file.md

```diff
@@ -292,61 +336,42 @@ In a `portable` target, the package entries may have an additional `subtargets`
                 "compile": {
                     "ref/netstandard1.5/System.Data.SqlClient.dll": {}
                 },
-                "subtargets": {
-                    "runtime": {
-                        "runtimes/unix/lib/netstandard1.5/System.Data.SqlClient.dll": { "rid": "unix" },
-                        "runtimes/win7-x64/lib/netstandard1.5/System.Data.SqlClient.dll": { "rid": "win7-x64" },
-                        "runtimes/win7-x86/lib/netstandard1.5/System.Data.SqlClient.dll": { "rid": "win7-x86" }
+                "runtimeTargets": {
```

Comment:
@schellap In order to unify this format with how NuGet is producing the data (/cc @emgarten), we've changed the layout of this section just slightly. Basically, we've flattened out the child nodes of `subtargets` and renamed it to `runtimeTargets`. The behavior is the same, and I've updated the resolution algorithm below to correspond to these changes.